Dockerfile

```diff
@@ -1,43 +1,45 @@
-FROM debian:buster
+ARG ALPINE_VERSION
+ARG NGINX_VERSION
+
```

Comment:
I think it would be also great to  have Docker  Image labels with versions of AiO and Nginx at least
https://docs.docker.com/engine/reference/builder/#label

Reply:
Added these